pkg/storaged/fsys-tab.jsx

```diff
@@ -403,26 +437,37 @@ export function mounting_dialog(client, block, mode, forced_options) {
         Title: cockpit.format(mode_title[mode], old_dir),
         Fields: fields,
         Teardown: teardown,
+        update: function (dlg, vals, trigger) {
+            if (trigger == "at_boot")
+                dlg.set_options("at_boot", { explanation: mount_explanation[vals.at_boot] });
+        },
         Action: {
             Title: mode_action[mode],
             action: function (vals) {
                 if (mode == "unmount") {
                     return do_unmount();
                 } else if (mode == "mount" || mode == "update") {
                     let opts = [];
-                    if ((mode == "update" && !is_filesystem_mounted) || vals.mount_options.never_auto)
+                    if ((mode == "update" && !is_filesystem_mounted) || vals.at_boot == "never")
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221115-130151-a219e579-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)